bcauth/templatetags/helpers.py

```diff
@@ -48,12 +50,6 @@ def provider_login_url(
     return provider.get_login_url(request, **query)
 
 
-env = {
-    'globals': {
-        'get_providers': get_providers,
-        'get_social_accounts': get_social_accounts,
-        'provider_login_url': provider_login_url,
-        'providers_media_js': providers_media_js,
```

Comment:
It looks like ``provider_login_url`` and ``providers_media_js`` are dropped from globals, which is breaking the login view at /accounts/login